.changeset/gentle-berries-pretend.md

```diff
@@ -0,0 +1,14 @@
+---
+"web": patch
+"@sovryn/onboard-core": patch
+"@sovryn/onboard-hw-common": patch
+"@sovryn/onboard-ledger": patch
+"@sovryn/onboard-react": patch
+"@sovryn/onboard-trezor": patch
+---
+
+SOV-438: added trezor module
+also these fixes:
```

Comment:
```suggestion
also these fixes:

```

Reply:
needs for prettier format

Reply:
it's actually auto generated files and will be automatically deleted from git once package is published (it's used by bot/cli to prepare changelogs and bump versions correctly)